### ISSUE TITLE
problem about model prediction

### ISSUE THREAD

**Issue by rezazad68** [mhjabreel/CharCnn_Keras#1]
hello, at first, thank for your contribution  and nice work

I got a problem about model prediction, just trained model with 10 epochs, after training model, when I used model.predict on whole validation set I got  all same value for each sample of validation set (returned value for each class on each sample of validation is : 0.250, 0.251, 0.250, 0.250 ) 
does it  right value? actually it assign any new sample to second class.

it would be great if u can help me
with regards

**Comment by FrancisChng** [comment]
I have some similar question
But I don't think I got the right way to use this model.

**Comment by royaljain** [comment]
Too late I think but if someone else stumbles upon this, here's what solved this for me -

Initially the values are pretty low and THresholdRelu layer clips them back to 0 and hence we get same output for everything

**Comment by chaitjo** [comment]
@royaljain What did you do to solve the problem? A different activation function to the ThresholdReLU?
